### DISCUSSION TITLE
Fallback for `:webp` modifier to prevent system crashes

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#4572] [Feature Requests]
### Suggest an idea for ExpressionEngine

In the development environment, when using the `:webp` modifier on images not present on the local server, the system throws the following exception:

```
Exception Caught
Cannot create a temp file from path: Mzk2L2VkaXRpb25zLzI0NjUvZmlsbXMvMTAyNTI3Ny9hY2Nlc3Nvcmllcy8xNTQwOTk2LnBuZw__.png
ee/ExpressionEngine/Library/Filesystem/Filesystem.php:1034
```

Please, add a fallback for cases where image files are missing on the server. Syncing all images locally is unrealistic and impractical for larger sites (currently >70GB for me). 

This would prevent dev crashes.

**Comment by intoeetive** [comment]
Converted into issue as it looks rather as a bug https://github.com/ExpressionEngine/ExpressionEngine/issues/4617
